GenderPayGap.WebUI/Views/AccountCreation/CreateUserAccount.cshtml

```diff
@@ -33,8 +33,7 @@
         </p>
         <p class="govuk-body">
             If you already have a user account, please
-            @Html.ActionLink("sign in", nameof(OrganisationController.ManageOrganisations),
-                "Organisation").
+            @(Html.ActionLink("sign in", "ManageOrganisationsGet", "ManageOrganisations")).
```

Comment:
Can you change this to be
```cshtml
<a href="@(Url.Action("ManageOrganisationsGet", "ManageOrganisations"))"
   class="govuk-body">
    sign in
</a>
```
At the moment, this link doesn't have the correct class (which mainly affects the tabbed focus colour)